visualization/app/codeCharta/ui/nodeContextMenu/nodeContextMenu.component.ts

```diff
@@ -109,7 +117,7 @@ export class NodeContextMenuController
 		angular.element(this.$element[0].children[0]).css("left", x + "px")
 	}
 
-	public hideNodeContextMenu(mousePosition = new Vector2(-1, -1)) {
+	public onHideNodeContextMenu(mousePosition = new Vector2(-1, -1)) {
```

Comment:
This function is now the implementation of an interface. Please move this function to the top, like we do with `onBlacklistChanged` etc